pkgs/by-name/se/seabios/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchgit
+, acpica-tools
+, python3
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "seabios";
+  version = "1.16.3";
+
+  src = fetchgit {
+    url = "https://git.seabios.org/seabios.git";
+    rev = "rel-${finalAttrs.version}";
+    hash = "sha256-hWemj83cxdY8p+Jhkh5GcPvI0Sy5aKYZJCsKDjHTUUk=";
+  };
+
+  outputs = [ "out" "doc" ];
+
+  nativeBuildInputs = [ python3 ];
+
+  buildInputs = [ acpica-tools ];
+
+  strictDeps = true;
+
+  hardeningDisable = [ "pic" "stackprotector" "fortify" ];
+
+  postConfigure = ''
+    cp ${./config.txt} .config
```

Comment:
Why must the file be separate rather inlined?

Reply:
Not a "must" but a preference. I believe it is cleaner.
The other possibility is to generate it via some converter like `toINI`.